### PR TITLE
feat: Make OCR default plugin work

### PR DESCRIPTION
**Pull request by wzyforgit**
1. Add OCR default plugin details
2. Reorganize debian files and cmake projects, dtkocr is a standalone package now
3. One known issue is dtkocr's .qch file is not ready now, it cannot be installed

Log: Make OCR default plugin work

### CODE REVIEW

**Review by xzl01 — COMMENTED** [review]
+1

**Comment by toberyan** [comment]
+1

File: src/ocr/interface/docr.cpp [hunk 1]
```diff
@@ -15,23 +22,49 @@ DOcrPrivate::DOcrPrivate(DOcr *parent)
     ;
 }
 
-DOcr::DOcr(QObject *parent)
-    : QObject(parent), d_ptr(new DOcrPrivate(this))
+bool DOcrPrivate::isCompatible()
+{
+    ;
```
Comment (Groveer) [inline src/ocr/interface/docr.cpp]: 这里是都没实现么

File: src/ocr/interface/docr.cpp [hunk 2]
```diff
@@ -15,23 +22,49 @@ DOcrPrivate::DOcrPrivate(DOcr *parent)
...
+void DOcrPrivate::resetPlugin()
+{
+    ;
+}
+
+QString DOcrPrivate::currentModuleDir()
 {
+    Dl_info dlInfo;
+    int rc;
+    rc = dladdr(this, &dlInfo);
```
Comment (Groveer) [inline src/ocr/interface/docr.cpp:39]: rc有啥用？

File: src/ocr/interface/docr.cpp [hunk 3]
```diff
@@ -15,23 +22,49 @@ DOcrPrivate::DOcrPrivate(DOcr *parent)
...
+
+    QString modelFullPath = dlInfo.dli_fname;
```
Comment (Groveer) [inline src/ocr/interface/docr.cpp]: const &

**Review by toberyan — DISMISSED** [review]

File: src/ocr/interface/docr.cpp (hunk 1 above)
Comment (wzyforgit) [reply src/ocr/interface/docr.cpp]: 这些是插件管理功能相关，暂未实现，现在已加上返回值，防止误调用而崩溃

File: src/ocr/interface/docr.cpp (hunk 2 above)
Comment (wzyforgit) [reply src/ocr/interface/docr.cpp:39]: 判断dlInfo值是否有效，已加上判断语句

File: src/ocr/interface/docr.cpp (hunk 3 above)
Comment (wzyforgit) [reply src/ocr/interface/docr.cpp]: 已加上

**Review by Groveer — COMMENTED** [review]
+1

**Review by Groveer — APPROVED** [review]

**Pull request merged**